README.md

```diff
@@ -28,7 +28,7 @@ If you would like to make changes to the SilverStripe core codebase, we have an
 
 ## Links ##
 
- * [Server Requirements](https://doc.silverstripe.org/framework/en/installation/server-requirements)
+ * [Server Requirements](https://docs.silverstripe.org/en/4/getting_started/server_requirements/)
```

Comment:
You could use a URL without the version number in it, as it was previously (so the new one would be https://docs.silverstripe.org/en/getting_started/server_requirements/) so that it automatically redirects to the latest version.

Reply:
Ideal, thanks :)